pegen/pegen.c

```diff
@@ -1690,7 +2595,10 @@ concatenate_strings(Parser *p, asdl_seq *strings)
     PyObject *u_kind = NULL;
     int kind_unicode = 0;
     PyObject *final_str = NULL;
+    return first;
```

Comment:
This is temporary until the rest of the f-string code is correct. It allows executing the f-string test without worrying of concatenation.